packages/handlebars/package.json

```diff
@@ -22,5 +22,8 @@
   },
   "peerDependencies": {
     "@frctl/fractal": "^1.1.0"
+  },
+  "devDependencies": {
```

Comment:
can you rebase this on master? i added the devDeps to this file in #616, but in a slightly different location - just so this won't start conflicting

Reply:
Done

Reply:
looks like this is still doubled

Reply:
Should be ok now